omod/src/main/web/dashboardwidgets/programstatus/programstatus.controller.js

```diff
@@ -738,22 +735,9 @@ export default class ProgramStatusController {
         this.$window.location.reload();
     }
 
-    dateWithoutTime(date) {
-        return moment(date).format('YYYY-MM-DD');
-    }
-
-    toUTCDate(dateString) {
```

Comment:
Note that this toUTCDate function was added to fix a similar issue to this issue (see RA-1774) however, as far as I can tell, the assumption this function was making was that the server dates are always in UTC, which may not be the case)